applications/services/gui/modules/number_input.h

```diff
@@ -0,0 +1,75 @@
+/**
+ * @file number_input.h
+ * GUI: Integer string keyboard view module API
+ */
+
+#pragma once
+
+#include <gui/view.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Number input anonymous structure  */
+typedef struct NumberInput NumberInput;
+
+/** callback that is executed on save button press */
+typedef void (*NumberInputCallback)(void* context);
+
+/** callback that is executed when byte buffer is changed */
+typedef void (*NumberChangedCallback)(void* context);
```

Comment:
Pass number or text to callback.
Also what's meaning of this callback? does it works as validator? In that case it should return something to signal validation results.

Reply:
You're right, this is not needed. Will be removed